### PR TITLE
Initial code

### PR DESCRIPTION
**Pull request by sreynen**
As this is the initial code, see README.md for description.

### CODE REVIEW

File: import_new_files.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    """
+    last_run = get_psql_results(args)[0].get('last_run')
+    last_run_file = last_run.strftime('%Y-%m-%d-%H-%M-00')
+    files = [file.split('/') for file in s3_file_list(args) if file.split('/')[0] > last_run_file]
+    files.sort(key=lambda file:file[0])
+    args.NO_RESULTS = True
+    if len(files):
+        for file in files:
+            args.DB_QUERY = """
+            COPY tao.vote_dot_org_events
```
Comment (schuyler1d) [inline import_new_files.py]: maybe the table name should also be an arg/variable?

File: import_new_files.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            FROM 's3://%s/%s/%s'
+            region AS '%s'
+            csv acceptinvchars
+            access_key_id '%s'
+            secret_access_key '%s';
+            """ % (
+                args.S3_BUCKET,
```
Comment (schuyler1d) [inline import_new_files.py]: generally it's best practice to make these query parameters rather than do python string interpolation.

**Review by schuyler1d — APPROVED** [review]
some optional nits.  Also would like a comment on the run-rate again -- why "2 hours" e.g.

Also, will this query create dupes in the table?  Parsons has an upsert pattern that copies to a temp table and then deletes existing records:
https://github.com/move-coop/parsons/blob/master/parsons/databases/redshift/redshift.py#L675

File: import_new_files.py (hunk 2 above)
Comment (sreynen) [reply import_new_files.py]: Updated the code to pass everything as a query param.

**Comment by sreynen** [comment]
I addressed the inline comments with the latest commit. To answer the other questions: the exports only include events since the last export, so tracking the last imported file should be enough to avoid duplicate records. And the suggested frequency is 2 hours just because that's how often they do the export for us. But that's also configurable.

**Pull request merged**